### ISSUE TITLE
[Android] emulator:true & jailbroken:true is wrong for android out of memory crashes

### ISSUE THREAD

**Issue by tayl0r** [bugsnag/bugsnag-unity#133]
### Description
I am looking at Android `SIGSEGV: Segmentation violation (invalid memory reference)` errors in Bugsnag. When you go into the Device tab, it says `emulator = true` and `jailbroken = true`. That is not correct however, as I'm reproducing this crash on a non-jailbroken non-emulator device.

### Issue
`emulator` and `jailbroken` should be correct

### Environment
Unity Android

* bugsnag-unity version: 4.2.0
* Unity version: 2017.4.14f1
* Operating system name and version: building with OSX
* Target platform names and versions: Android 8.0 on HUAWEI ANE-LX2 (P20 Lite)
* Initializing bugsnag via the Unity UI or in code? UI

### Example code snippet

```
// run this method and it will crash your android device. start it up again and you should get a SIGSEGV error in Bugsnag.
// Then look at the Device tab and check what it says for Emulator and Jailbroken.
public void MemoryCrash() {
	StartCoroutine(MemoryCrashCor());
}

IEnumerator MemoryCrashCor() {
	Debug.Log("MemoryCrashCor");
	System.Text.StringBuilder sb = new System.Text.StringBuilder();
	List<string> list = new List<string>();
	for (int i = 0; i < int.MaxValue; ++i) {
		if (i % 100 == 0) {
			yield return 1;
		}
		try {
			sb.AppendFormat("!! {0},{1},{2},{3} !!\n", i, i * 2, Mathf.Pow((float)i, 2f), int.MaxValue - i);
			list.Add(sb.ToString());
		} catch {
			Debug.Log("MemoryCrashCor .... BOOOM!!");
		}
	}
}
```

**Comment by tayl0r** [comment]
![image](https://user-images.githubusercontent.com/323663/53388251-35cf1f00-393f-11e9-9734-13d68c09903b.png)

the `totalMemory` value is also wrong. There is no way this device has only 256 MB of RAM.

**Comment by fractalwrench** [comment]
Thanks for the report @tayl0r - we've scheduled this issue for investigation and will track it in this issue: https://github.com/bugsnag/bugsnag-android/issues/447

**Comment by snmaynard** [comment]
This has been fixed in 4.13.0 on android -> https://github.com/bugsnag/bugsnag-android/releases/tag/v4.13.0 Thanks for reporting it to us! I believe the fix is on its way to Unity soon cc @fractalwrench
